panel/tests/widgets/test_chat.py

```diff
@@ -1090,15 +1090,15 @@ def test_reset_on_send(self, chat_interface):
         assert chat_interface.active_widget.value == ""
 
     def test_reset_on_send_text_area(self, chat_interface):
```

Comment:
Why take an input `chat_interface` and then override it in the next line?

Reply:
Initially I couldn't get it to work, but with Philipp's changes, it works so I reverted it.